docs/source/tutorial_external_solver.rst

```diff
@@ -1,6 +1,8 @@
 Tutorial: Binding an external PDE solver to pyMOR
 =================================================
 
+.. include:: jupyter_init.txt
```

Comment:
Did you check if this has an effect? I'm using a bash kernel first and only switch to Python afterwards.

Reply:
Actually no, it does not seem to have an effect. Probably should be moved to the point where the Python kernel has been activated?

Reply:
That should do it, yes.